mapie/regression.py

```diff
@@ -69,6 +68,9 @@ class MapieRegressor(BaseEstimator, RegressorMixin):
           - ``sklearn.model_selection.LeaveOneOut`` (jackknife),
           - ``sklearn.model_selection.KFold`` (cross-validation),
           - ``subsample.Subsample`` object (bootstrap).
+        - ``"split"``, does not involve cross-validation but a division
```

Comment:
remove ""

Reply:
Why removing "" if we do the same for '"prefit"` (line 74)?